packages/hoppscotch-common/src/services/spotlight/searchers/collections.searcher.ts

```diff
@@ -0,0 +1,315 @@
+import { Service } from "dioc"
+import {
+  SpotlightSearcher,
+  SpotlightSearcherResult,
+  SpotlightSearcherSessionState,
+  SpotlightService,
+} from "../"
+import { Ref, computed, effectScope, markRaw, ref, watch } from "vue"
+import { getI18n } from "~/modules/i18n"
+import MiniSearch from "minisearch"
+import {
+  graphqlCollectionStore,
+  restCollectionStore,
+} from "~/newstore/collections"
+import IconFolder from "~icons/lucide/folder"
+import RESTRequestSpotlightEntry from "~/components/app/spotlight/entry/RESTRequest.vue"
+import GQLRequestSpotlightEntry from "~/components/app/spotlight/entry/GQLRequest.vue"
+import { createNewTab } from "~/helpers/rest/tab"
+import { getTabRefWithSaveContext } from "~/helpers/rest/tab"
+import { currentTabID } from "~/helpers/rest/tab"
+import {
+  HoppCollection,
+  HoppGQLRequest,
+  HoppRESTRequest,
+} from "@hoppscotch/data"
+import { setGQLSession } from "~/newstore/GQLSession"
+import { cloneDeep } from "lodash-es"
+import { hoppWorkspaceStore } from "~/newstore/workspace"
+import { changeWorkspace } from "~/newstore/workspace"
+
+/**
+ * A spotlight searcher that searches through the user's collections
+ *
+ * NOTE: Initializing this service registers it as a searcher with the Spotlight Service.
+ */
+export class CollectionsSpotlightSearcherService
+  extends Service
+  implements SpotlightSearcher
+{
+  public static readonly ID = "COLLECTIONS_SPOTLIGHT_SEARCHER_SERVICE"
+
+  private t = getI18n()
+
+  public searcherID = "collections"
+  public searcherSectionTitle = this.t("tab.collections")
```

Comment:
Change the spotlight section title from "Collections" to "My Collections" or ideally "My Workspace > Collections".

<img width="673" alt="Screenshot 2023-08-18 at 5 22 06 PM" src="https://github.com/hoppscotch/hoppscotch/assets/10395817/fb401af1-79aa-4768-ae2c-bc2eca610fc9">